include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -190,14 +191,23 @@ MjbotsControlLoop<log_type, input_type, robot_type>::MjbotsControlLoop(std::shar
     joints_moteus.push_back(
         std::dynamic_pointer_cast<kodlab::mjbots::JointMoteus>(j));
   }
+  // Open loop check
+  for(const auto & j : joints_moteus){
+      if(!options.open_loop && j->is_open_loop()){
+        LOG_FATAL("Open loop option not set, not receiving position and/or velocity");
```

Comment:
Lets make this a little more clear. Maybe:

`ControlLoopOptions has open_loop=false, but joint moteus QueryCommand struct does not query position and or velocity data. `